Python_and_Git/task_7_is_id.py

```diff
@@ -0,0 +1,14 @@
+x = int(500)
+y = int(500)
+print(x is y)
+p = int(10)
+q = int(10)
+print (p is q)
+print (id (p))
+print (id (q))
+print (id(p) == id(q))
+p += 0
+print (id (p))
+print (id (q))
+print (id(p) == id(q))
+print (p is q)
```

Comment:
No need to specify type of variables)
```suggestion
x = 500
y = 500
print(x is y)
p = 10
q = 10
print(p is q)
print(id (p))
print(id (q))
print(id(p) == id(q))
p += 0
print (id (p))
print (id (q))
print (id(p) == id(q))
print (p is q)
```